src/neo/Network/P2P/Peer.cs

```diff
@@ -141,7 +167,8 @@ private void OnStart(ChannelsConfig config)
             MaxConnections = config.MaxConnections;
             MaxConnectionsPerAddress = config.MaxConnectionsPerAddress;
 
-            timer = Context.System.Scheduler.ScheduleTellRepeatedlyCancelable(0, 5000, Context.Self, new Timer(), ActorRefs.NoSender);
+            // schedule time to trigger `OnTimer` event every TimerMillisecondsInterval ms
+            timer = Context.System.Scheduler.ScheduleTellRepeatedlyCancelable(0, TimerMillisecondsInterval, Context.Self, new Timer(), ActorRefs.NoSender);
```

Comment:
I think such changes actually reduce readability.

Reply:
Let's revert this one. It, in fact, should had not come here.
It was from a previous PR and was merged for some reason.